Resources/views/CRUD/base_history.html.twig

```diff
@@ -33,16 +33,35 @@ file that was distributed with this source code.
                         </thead>
                         <tbody>
                         {% for revision in revisions %}
-                            <tr class="{% if (currentRevision != false and revision.rev == currentRevision.rev) %}current-revision{% endif %}">
+                            <tr class="{{ classlist({
+                                'sonata-ba-revisions__revision': true,
+                                'sonata-ba-revisions__revision--is-current': currentRevision != false and revision.rev == currentRevision.rev
```

Comment:
Where : as soon as you need to use the horizontal scrollbar on github would be great. Having lines shorter than 120 chars should be the goal IMO.

How : in this instance, like this maybe ? :  
                                'sonata-ba-revisions__revision--is-current': 
                                    currentRevision != false 
                                    and 
                                    revision.rev == currentRevision.rev